components/motor/gpio/encoded.go

```diff
@@ -88,8 +88,8 @@ func newEncodedMotor(
 	}
 
 	if em.rampRate == 0 || em.rampRate > 0.5 {
-		em.rampRate = 0.05 // Use a conservative value by default.
 		em.logger.Warnf("setting ramp rate to default value of 0.05 instead of %v", em.rampRate)
+		em.rampRate = 0.05 // Use a conservative value by default.
```

Comment:
this I just happened to see when I was testing and before it would always print out "setting ramp rate to default value of 0.05 instead of 0.05" so I changed it